SpatialGDK/Source/SpatialGDK/Public/Utils/ComponentReader.h

```diff
@@ -26,12 +26,43 @@ class ComponentReader
 							  USpatialActorChannel& Channel, bool& bOutReferencesChanged);
 
 private:
-	void ApplySchemaObject(Schema_Object* ComponentObject, UObject& Object, USpatialActorChannel& Channel, bool bIsInitialData,
-						   const TArray<Schema_FieldId>& UpdatedIds, Worker_ComponentId ComponentId, bool& bOutReferencesChanged);
+	enum EProcessFieldType
```

Comment:
Nit: prefer `enum class` (doesn't make a huge difference for the current use case, but it's generally nicer because the value names are local to the enum and prevents implicit conversions to and from ints)

Reply:
Thanks! I didn't know it made that difference!